build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '4.4.1'
+version '4.4.2-SANPSHOT'
```

Comment:
SNAP :)

Reply:
I think you need to bump up the Y part of the version (X.Y.Z)

Reply:
yeah - this is a good point - maybe should bump major version X (semver = incompatible changes...is it? if not then fine)

Reply:
This is new backward compatible functionality , so don't need to bump X.